pkg/client/unversioned/request.go

```diff
@@ -244,11 +244,10 @@ func (r *Request) AbsPath(segments ...string) *Request {
 	if r.err != nil {
 		return r
 	}
-	if len(segments) == 1 {
+	r.path = path.Join(r.baseURL.Path, path.Join(segments...))
```

Comment:
This is inconsistent with line 128-- there, r.path only has the api part. Here, it will also include the proxy prefix. Won't this result in doubling the proxy prefix?